hw5/FindMaxMinValue.java

```diff
@@ -0,0 +1,27 @@
+package hw4;
+
+public class FindMaxMinValue {
+
+    //2. Написати програму, яка знаходить мінімальне і максимальне число в масиві.
+    static void findMaxMinValue(int[] array) {
```

Comment:
Looks like you do not check the last element in array

Reply:
initialized var max and min as first var from array and cycle started with "int i = 1"

Reply:
I have no idea why this comment here) It was for some other person.
I am sorry!